source/core/simulation_control.cc

```diff
@@ -48,6 +55,116 @@ SimulationControl::is_output_iteration()
   return (get_step_number() % output_frequency == 0);
 }
 
+void
+SimulationControl::update_assembly_method()
+{
+  if (iteration_number <= 1 &&
+      method == Parameters::SimulationControl::TimeSteppingMethod::bdf2 &&
+      bdf_start_method ==
+        Parameters::SimulationControl::BDFStartupMethods::multiple_step_bdf)
+    {
+      assembly_method = Parameters::SimulationControl::TimeSteppingMethod::bdf1;
+      set_current_time_step(initial_time_step * startup_timestep_scaling);
```

Comment:
One thing we could consider would be to abandon this startup_timestep_scaling parameter in a next PR and just use the regular time step and live with it.